src/rpdk/core/contract/resource_client.py

```diff
@@ -234,6 +235,13 @@ def make_request(desired_resource_state, previous_resource_state, **kwargs):
     def generate_token():
         return str(uuid4())
 
+    @staticmethod
+    def assert_time(start_time, end_time, action):
+        timeout_in_seconds = 60
+        if action in (Action.READ, Action.LIST):
+            timeout_in_seconds = 30
+        assert end_time - start_time <= timeout_in_seconds
```

Comment:
can we add some kind of meaningful message if the handler fails this assertion?

Reply:
do you mean Log before we assert?

Reply:
what will the customer see when this assertion fails?  I'm guessing it will just say "assertion error: line 243" with no useful info.  I'm pretty sure assert takes a message parameter, or if not maybe logging will work? As long as the customer can clearly see why their test failed

Reply:
makese sense.

Reply:
pytest is actually pretty detailed and prints out entire stack trace and values

Reply:
yeah but will it tell them that their handler timed out/took too long? as opposed to just saying "70 > 60"